.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build
+
+on:
+  push:
+    branches: [ "6.0.6-ded4" ]
+  pull_request:
+    branches: [ "6.0.6-ded4" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.18]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
```

Comment:
why not have a build/test step? why just build?

Reply:
Currently build does : npm run test && npm run clean:dist && npm run version && npm run build:prod && npm run build:dev

Reply:
Currently build does : npm run test && npm run clean:dist && npm run version && npm run build:prod && npm run build:dev